fb_instant_articles.module

```diff
@@ -18,6 +18,16 @@ function fb_instant_articles_menu() {
     'file path' => drupal_get_path('module', 'system'),
   );
 
+  $items['admin/config/services/fb-instant-articles/settings'] = array(
```

Comment:
Oh sure. That does sound reasonable… the link for the base settings (right now just the page ID) is to be found on that system blocks page anyway). Will do now